terrain_generator:
  ros__parameters:
    terrain_type: 'forest'  # Options: 'forest', 'mountains', 'plains'
    space_dim: [100.0, 100.0, 50.0]  # [x, y, z] in meters
    start_pos: [0.0, 0.0, 0.0]  # Start position to avoid obstacles
    
    # Forest parameters
    forest:
      grid_size: 10
      radius_range: [0.5, 1.5]  # Tree radius range (m)
      height_range: [5.0, 15.0]  # Tree height range (m)
      density: 0.7  # Probability of tree in each grid cell (0-1)
    
    # Mountains parameters
    mountains:
      num_peaks: 15
      base_size_range: [3.0, 8.0]  # Base size range (m)
      height_range: [10.0, 30.0]  # Peak height range (m)
    
    # Plains parameters
    plains:
      num_obstacles: 10
      obstacle_types: ['bush', 'rock']  # Options: 'bush', 'rock', 'tree'
